pom.xml

```diff
@@ -210,6 +210,16 @@
         <artifactId>javax.servlet.jsp.jstl-api</artifactId>
         <version>1.2.2</version>
       </dependency>
+	  <dependency>
+		 <groupId>javax.servlet</groupId>
+		 <artifactId>jstl</artifactId>
+		 <version>1.2</version>
+	  </dependency>
+	  <dependency>
+	    <groupId>taglibs</groupId>
+	    <artifactId>standard</artifactId>
```

Comment:
```
<dependency>
    <groupId>org.apache.taglibs</groupId>
    <artifactId>taglibs-standard-impl</artifactId>
    <version>1.2.5</version>
    <scope>runtime</scope>
</dependency>
```

Reply:
Upd: https://mvnrepository.com/artifact/org.apache.taglibs/taglibs-standard-impl/1.2.5 now shows it has a CVE.